phoenix-core/src/it/java/org/apache/phoenix/end2end/index/BaseIndexWithRegionMovesIT.java

```diff
@@ -1709,6 +1709,7 @@ public void testLastDDLTimestampOnAsyncIndexes() throws Exception {
 
             // run the index MR job.
             IndexToolIT.runIndexTool(false, TestUtil.DEFAULT_SCHEMA_NAME, tableName, indexName);
+            TestUtil.waitForIndexState(conn, fullIndexName, PIndexState.BUILDING);
```

Comment:
It seems this is the problem, it should be `PIndexState.ACTIVE`

Reply:
@haridsv i missed this diff while reviewing, could you please fix this?

Reply:
Thanks for catching and sorry about the mistake.

Reply:
No worries at all, Thank you for fixing it